tests/media/server/gstplayer/genericPlayer/GstGenericPlayerPrivateTest.cpp

```diff
@@ -653,8 +653,9 @@ TEST_F(GstGenericPlayerPrivateTest, shouldUpdateAudioCaps)
     EXPECT_CALL(*m_gstWrapperMock, gstCapsSetSimpleIntStub(&dummyCaps2, CharStrMatcher("rate"), G_TYPE_INT, kSampleRate));
     EXPECT_CALL(*m_gstWrapperMock,
                 gstCapsSetSimpleIntStub(&dummyCaps2, CharStrMatcher("channels"), G_TYPE_INT, kNumberOfChannels));
-    EXPECT_CALL(*m_glibWrapperMock, gMemdup(arrayMatcher(*kCodecData), kCodecData->size())).WillOnce(Return(memory));
-    EXPECT_CALL(*m_gstWrapperMock, gstBufferNewWrapped(memory, kCodecData->size())).WillOnce(Return(&buf));
+    EXPECT_CALL(*m_glibWrapperMock, gMemdup(arrayMatcher(kCodecData->data), kCodecData->data.size()))
+        .WillOnce(Return(memory));
```

Comment:
Add a test or change one of the current tests to use CodecDataType::STRING, to make sure that it is successfully propagated to gstreamer?

Reply:
will do